great_expectations_cloud/agent/models.py

```diff
@@ -109,6 +109,7 @@ class GenerateSchemaChangeExpectationsEvent(EventBase):
     )
     datasource_name: str
     data_assets: Sequence[str]
+    # TODO: Remove data_asset_to_expectation_suite_name from the event
```

Comment:
Is this a ticket? I'm in the one that added this parameter, so I'd be happy to do the removal work too :)

Reply:
It is! I forgot to mark it "TODO": https://greatexpectations.atlassian.net/browse/ZELDA-1166